core/modd.conf

```diff
@@ -1,3 +1,10 @@
+# For .templ, .sql, and .proto files
+**/*.templ **/*.sql **/*.proto {
+    prep: make gen
+}
+
+# For .go files
 **/*.go {
-    prep: make dev
+    prep: make build-amd64
+    daemon: clear && make sandbox-node
 }
```

Comment:
Only run codegen when templ (html), sql (sqlc), or proto files change. Should this modify or create any new go files the next **.*.go will kick in and rebuild the binary and rebuild the sandbox container. We could consider mounting the binary to the sandbox container and just restarting instead.